src/kontrol/foundry.py

```diff
@@ -266,6 +277,28 @@ def _filter_constraints_by_simplification(
         new_cterm = CTerm.from_kast(set_cell(cterm.kast, 'K_CELL', KSequence(subst['###CONTINUATION'])))
         return Step(CTerm(new_cterm.config, new_constraints), 1, (), ['cheatcode_forget'], cut=True)
 
+    def _exec_console_log_custom_step(self, subst: Subst, cterm: CTerm, _c: CTermSymbolic) -> KCFGExtendResult | None:
+        selector_token = subst['###SELECTOR']
+        data = subst['###DATA']
+        assert type(selector_token) is KToken
+
+        try:
+            if type(data) is KToken:
+                selector = int(selector_token.token)
+                print(data)
```

Comment:
Is this a debugging statement that should be removed by any chance?
```suggestion
                print(data)
```

Reply:
Great catch! Thanks!